tests/scalers/redis/redis_cluster_streams_lag/redis_cluster_streams_lag_test.go

```diff
@@ -0,0 +1,235 @@
+//go:build e2e
```

Comment:
Rename to: tests/scalers/redis/redis_cluster_streams/redis_cluster_streams_lag_test.go